stable/mssql-linux/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: SQL Server 2017 Linux Helm Chart
 name: mssql-linux
-version: 0.6.5
+version: 0.6.6
```

Comment:
Adding features here, so this should be minor version bumped. As this chart is in stable, I would actually do a bump to 1.0.0

Reply:
Updated to 1.0.0